sys/include/net/gnrc.h

```diff
@@ -155,26 +155,46 @@
  * Transmitting Packets
  * --------------------
  *
- * To relay the @ref net_gnrc_pkt to interested threads the @ref
- * net_gnrc_netapi offers a dispatch functionality. The following example
+ * A packet is transmitted by relaying it to threads interested in handling (and
+ * dispatching) packets of its type. To do this, the @ref
+ * net_gnrc_netapi offers a dispatch functionality called @ref
+ * gnrc_netapi_dispatch_send().
+ *
+ * The following example
  * sketches the usage and assumes a valid @ref net_gnrc_pkt named `pkt`.
  *
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ {.c}
+ * gnrc_pktsnip_t *pkt;
+ *
+ * pkt = gnrc_pktbuf_add(NULL, data, size, GNRC_NETTYPE_UNDEF);
+ * if (pkt == NULL) {
+ *      puts("Error: unable to copy data to packet buffer\n");
+ *      return;
+ * }
  * if (!gnrc_netapi_dispatch_send(GNRC_NETTYPE_UDP, 80, pkt)) {
```

Comment:
Maybe add a comment, that the own-age of `pkt` is given to the receiver threads by calling this function?